go/client/cmd_team_generate_seitan.go

```diff
@@ -88,4 +106,10 @@ func (c *CmdTeamGenerateSeitan) GetUsage() libkb.Usage {
 
 const teamGenerateSeitanDoc = `"keybase team generate-token" allows you to create a one-time use,
 expiring, cryptographically secure token that someone can use to join
-a team.`
+a team.
+
+Optionally, full name and phone number can be provided (using
+--fullname and --number flags) to label created token to make them
+easier to distinguish. Label data is encrypted and visible only to
+admins.
```

Comment:
I think we should make this mandatory, no?  So the UI has something to show...

Reply:
ok, both mandatory? or at least one should be entered?

Reply:
I talked it over with @malgorithms, it's OK for now as is.

Reply:
OK, thank you